### ISSUE TITLE
Add babel-plugin-transform-class-properties to neutrino-preset-react

### ISSUE THREAD

**Issue by heibor** [mozilla-neutrino/neutrino-dev#228]
The babel plugin babel-plugin-transform-class-properties is used to support some ES6+ syntax, such as property initializers:

```
class Video extends React.Component {
  static defaultProps = {
    autoPlay: false,
    maxLoops: 10,
  }
  static propTypes = {
    autoPlay: React.PropTypes.bool.isRequired,
    maxLoops: React.PropTypes.number.isRequired,
    posterFrameSrc: React.PropTypes.string.isRequired,
    videoSrc: React.PropTypes.string.isRequired,
  }
  state = {
    loopsRemaining: this.props.maxLoops,
  }
}
```

Some features can promote the devlopment using react. Maybe it should be added into neutrino-preset-react.

**Comment by constgen** [comment]
I do this any way on my projects with React
```
  config.module.rule('compile').use('babel').tap(options => {
    const decoratorsPlugin = require.resolve('babel-plugin-transform-decorators-legacy');
    const classPropertiesPlugin = require.resolve('babel-plugin-transform-class-properties');
    options.plugins.unshift(decoratorsPlugin, classPropertiesPlugin);

    return options;
  });
```
So probably `babel-plugin-transform-class-properties` whould be a smart default.

**Comment by eliperelman** [comment]
I agree as well. Using class properties is common React projects, so let's add it.

**Comment by constgen** [comment]
@eliperelman can you prioritize this?

**Comment by eliperelman** [comment]
Resolved in #234.

**Comment by constgen** [comment]
I have reviewed the commit and seems that transformer is added to the development mode only. But should be attached to all modes in this array
```
plugins: [require.resolve('babel-plugin-transform-object-rest-spread')],
```
